example/demo.dart

```diff
@@ -37,14 +37,36 @@ main() {
   group('smallcheck', () {
     final sc = new SmallCheck(depth: 10);
     test('good', () => sc.check(goodProperty));
-    test('bad', () => sc.check(badProperty));
+
+    test('bad', () {
+      try {
```

Comment:
The purpose of `demo.dart` is to show that if you write unit tests that way, they will fail when your property is violated. The goal is for the user to run dart example/demo.dart and to observe the output on the console to get a feeling of what it is like to use the lib. So I'd rather not catch the exceptions.

Reply:
If I don't catch the error, then the test fails. I figured that was due to the unittest API at the time not having a way to expect an error to be thrown.

I tried `expect(sc.check(badProperty), throwsA(equals('falsified after 11 tests\n'
            '  argument 1: [true]\n'
            '  argument 2: [false]\n'
            ''));`

But I couldn't get it to work as expected (no pun intended).

Reply:
The point is that `demo.dart` is not a test of the `propcheck` library. Instead it is a file meant to showcase the library and its tests fail on purpose to showcase the error messages.

Reply:
Okay, then I think I did the try-catch before I realized enumerators need to be updated. Once that is done, I can retest this propcheck branch to ensure nothing broke (at least according to the tests).

Reply:
Okay, [this is the error that occurs when I run demo.dart](https://gist.github.com/Stargator/24a1c6503e8b9f620b0e481924712555). I added it to a gist with a version of the output from the master branch for comparison.